modules/videoio/src/plugin_capture_api.hpp

```diff
@@ -121,6 +121,27 @@ struct OpenCV_VideoIO_Capture_Plugin_API_v1_1_api_entries
         CV_OUT CvPluginCapture* handle);
 }; // OpenCV_VideoIO_Capture_Plugin_API_v1_1_api_entries
 
+struct OpenCV_VideoIO_Capture_Plugin_API_v1_2_api_entries
+{
+    /** @brief Open video capture with parameters
+
+    @param filename File name or NULL to use camera_index or buffer instead
+    @param camera_index Camera index (used if filename == NULL and buffer == NULL)
+    @param buffer Memory buffer pointer (used if filename == NULL)
```

Comment:
Do we really need to handle all 3 parameters for the source in a single method of API?
As we keep the previous version of the method, then we don't need to duplicate its functionality.

Reply:
Sorry foe delay. Changed to a method which accepts only buffer.

Wanted to add GStreamer backend but couldn't make it work for now.